version/version.go

```diff
@@ -33,12 +33,6 @@ func (version) Usage() string {
 }
 
 func (version) Execute(context extpoints.Context) bool {
```

Comment:
I would have sent this change under a different PR/code review since it's a different feature, but otherwise there is no issue here.